README.md

```diff
@@ -72,6 +72,7 @@ The community meeting calendar is available as an [iCal to subscribe to] (https:
   * Put a description of the topic in the welcome message
   * Set "Join the group" and "Post" to "Public"
 * Ask a repo maintainer to create a github label, if one doesn't already exist: area/foo
+* Ask organization maintainer to create a SIG-team within the [Kubernetes GitHub organization](https://github.com/orgs/kubernetes/teams)
```

Comment:
the standards for creating a SIG are outlined in README.md.  Basically, it's a loose consensus that some topic is not covered in existing SIGs and that there is a need to create a new one.  There is work going on in the kubernetes-pm group discussing this.  https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/kubernetes-pm/joQ_q0MaJ0w/hMdaklE-AgAJ

I think SIG teams is the plan.